integration-cli/docker_cli_service_logs_experimental_test.go

```diff
@@ -213,3 +198,51 @@ func (s *DockerSwarmSuite) TestServiceLogsFollow(c *check.C) {
 
 	c.Assert(cmd.Process.Kill(), checker.IsNil)
 }
+
+func (s *DockerSwarmSuite) TestServiceLogsTaskLogs(c *check.C) {
+	testRequires(c, ExperimentalDaemon)
+
+	d := s.AddDaemon(c, true, true)
+
+	name := "TestServicelogsTaskLogs"
+	replicas := 2
+
+	out, err := d.Cmd(
+		// create a service with the name
+		"service", "create", "--name", name,
+		// which has some number of replicas
+		fmt.Sprintf("--replicas=%v", replicas),
+		// which has this the task id as an environment variable templated in
+		"--env", "TASK={{.Task.ID}}",
+		// and runs this command to print exaclty 6 logs lines
+		"busybox", "sh", "-c", "for line in $(seq 0 5); do echo $TASK log test $line; done; sleep 100000",
+	)
+	c.Assert(err, checker.IsNil)
+	c.Assert(strings.TrimSpace(out), checker.Not(checker.Equals), "")
```

Comment:
I think this could easily pass when you don't expect it to. `out` is actually `Combined()`, so any error message would allow this to pass. Using `icmd.Expected{Out: ...}` is a much stricter assertion.